draft-irtf-cfrg-hpke.md

```diff
@@ -1475,6 +1474,28 @@ several features that a more high-level protocol might provide, for example:
   {{?I-D.ietf-tls-esni}} and {{?RFC8467}} for examples of protocol-specific
   padding policies.
 
+## Bidirectional Encryption {#bidirectional}
+
+As discussed in {{hpke-dem}}, HPKE encryption is unidirectional from sender
+to receiver. Applications that require bidirectional encryption can derive
+necessary keying material with the Secret Export interface {{hpke-export}}.
+The type and length of such keying material depends on the application use
+case.
+
+As an example, if an application needs AEAD encryption from receiver to
+sender, it can derive a key and nonce from the corresponding HPKE context
+as follows:
+
+~~~
+secret = context.Export("response", Nh)
+prk = Extract("", secret)
+key = Expand(prk, "key", Nk)
+nonce = Expand(prk, "nonce", Nn)
+~~~
```

Comment:
Instead of redoing an Extract + Expand iteration, I'd suggest a more lightweight example:

```suggestion
~~~
key = context.Export("response key", Nk)
nonce = context.Export("response nonce", Nn)
~~~
```
or if we can somehow express splitting of bitstrings in our pseudocode:

```suggestion
~~~
key, nonce = context.Export("response", Nk + Nn)
~~~
```